algebraic-gpu/test-vectors/mycircuit-input1.json

```diff
@@ -0,0 +1,2 @@
+{"a": 3, "b": 11}
```

Comment:
Can we hardcode those files in the source code accordingly to reduce the file scale?  same for the algebraic project